docs/modules/gcloud.md

```diff
@@ -36,3 +39,16 @@ Datastore mode | DatastoreEmulatorContainer | [gcr.io/google.com/cloudsdktool/g
 <!--codeinclude-->
 [Starting a Datastore Emulator container with a custom emulator image](../../packages/modules/gcloud/src/datastore-emulator-container.test.ts) inside_block:datastore5
 <!--/codeinclude-->
+
+### Cloud PubSub mode
+
+<!--codeinclude-->
+[Starting a Cloud PubSub Emulator container with the default image](../../packages/modules/gcloud/src/pubsub-emulator-container.test) inside_block:pubsub6
```

Comment:
Docs build is failing, this should be `test.ts`

Reply:
I have updated the page :)

Reply:
@cristianrgreco Any more changes needed? Love to stop using my fork of this package :)